README.md

```diff
@@ -31,71 +35,112 @@ Before you start writing _any_ code:
+
+### Minimum Requirements
 
-## Minimum Requirements
 However far you and your partner get, the application should have, at a minimum, the following features:
-- Seeded data in your database from the original CSV files
-- Multiple, related, Active Record models
+
+- Seeded data in your database from the original CSV files (see below)
+- Multiple related Active Record models
 - [RESTful routing](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/08-rails/mvc-and-restful-routing.md)
 - Deployed application to [Heroku](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/00-programming-fundamentals/11-deploying-to-heroku.md)
 - A "look and feel" that will make you and your partner happy!
 
-## Things to Keep in Mind
+### Seeding the Database
+
+We have provided sample data for your site in the form of CSV files. These files are located in the directory `db/seed_data/`. We have also provided a seed script, `db/seeds.rb`, to load these files into your database.
+
+`db/seeds.rb` assumes a few things about how your database is set up:
 
-1.  Try to include useful business logic in your model classes.
-1.  You can include nested routes, but avoid routes that are more than 2-levels deep.
-1.  Use Semantic HTML
-1.  Make good use of partial views
-1.  Don't forget validations! (can a trip exist without a driver or passenger...)
-1.  You may not be able to get all the user stories done.  **Prioritize!** 
+- There are `Driver`, `Passenger` and `Trip` models
+- These models are related in a certain way
+- The field names of these models match the column names in the CSV files
+
+This is the recommended way to configure this project, but not a hard requirement. If you've got a good reason to change the setup, feel free to edit `db/seeds.rb` accordingly.
+
+### Things to Keep in Mind
+
+1. Use git relentlessly
+1. Pair program on difficult problems and to share learning experiences
+1. Deploy early (Wednesday), deploy often.
+1. Business logic should be implemented in the model
+1. You will probably need nested routes, but avoid routes that are more than 2 levels deep
+1. Use Semantic HTML
+1. Make good use of partial views
+1. Regarding validations:
+    - Validations are useful for making sure your database records are sane, and you should make use of them on this project
+    - It is easy to validate too much! If it's not a hard requirement for your app to function, it probably shouldn't be a validation.
+    - If validations fail, let the user know why
 
 ## User Stories
 
+This project has a lot of user stories, and it is likely you and your partner won't get to all of them. Here is our prioritized list of what's important for this project, based on this week's learning goals:
+
+1. CRUD operations on models with relations
+    - Creating new trips
+1. Model validations and reporting errors to the user
+1. Deployment
+1. Basic layout of a complex Rails app
```

Comment:
Layout might get confused with HTML/CSS - could we adjust to "structure" instead?